python/vowpalwabbit/sklearn_vw.py

```diff
@@ -20,14 +20,17 @@
 
 
 class VW(BaseEstimator):
-    """Vowpal Wabbit Scikit-learn Base Estimator wrapper
-
-        Attributes
-        ----------
-        params : {dict}
-                 dictionary of model parameter keys and values
-        fit_ : {bool}
-               this variable is only created after the model is fitted
+    r"""
```

Comment:
Why do these need to become raw strings?

Reply:
This change will be useful when we will use Sphinx for auto docs. In Python docstrings, it will be necessary to escape any backslash characters so that they actually reach reStructuredText. The simplest way to do this is to use raw strings by adding the letter r in front of the docstring.
[https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html](https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html)

Reply:
I figured that the actual problem why it didn't appear in the docs was that it has no 'numpydoc' extension provided by sphinx.